tests/components/zwave/test_init.py

```diff
@@ -32,6 +30,23 @@ def mock_storage(hass_storage):
     """Autouse hass_storage for the TestCase tests."""
 
 
+@pytest.fixture
+def zwave_setup(hass):
+    """Zwave setup."""
+
+    async def _zwave():
```

Comment:
Why make this a method that is returned instead of just setting it up in the fixture?

Reply:
I also noticed that you call async_block_till_done() after calling this method. Consider including that inside this function.

Reply:
> Why make this a method that is returned instead of just setting it up in the fixture?

No particular reason, copied it from `zone` tests which used same logic. Thinking about it, it's not necessary to do so, `async_block_till_done` can be added and the whole `mock_connect` is not required here. Refactored it.